### PR TITLE
add workload config api

### LINKED ISSUE #6595: add reqmgr2 APIs updating splitting parameter and creating string format of workload config
Corresponding reqmgr APIs are

getting splitting parameter
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/HTTPFrontEnd/RequestManager/ReqMgrBrowser.py#L118

updating splitting parameter
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/HTTPFrontEnd/RequestManager/ReqMgrBrowser.py#L146

Workload condig
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/HTTPFrontEnd/RequestManager/ReqMgrBrowser.py#L267

### PR DESCRIPTION
**Pull request by ticoann**
fixes #6595 
Still need to add put request but making the pull request for the format agreement.

### CODE REVIEW

File: src/python/WMCore/ReqMgr/Service/Auxiliary.py [hunk 1]
```diff
@@ -23,6 +23,8 @@
 import WMCore.ReqMgr.Service.RegExp as rx
 from WMCore.REST.Format import JSONFormat
 
+from WMCore.WMSpec.WMWorkload import WMWorkloadHelper
```
Comment (ericvaandering) [inline src/python/WMCore/ReqMgr/Service/Auxiliary.py]: Does this class do anything (used anywhere)?

File: src/python/WMCore/ReqMgr/Service/Auxiliary.py [hunk 2]
```diff
@@ -23,6 +23,8 @@
...
+
 class HelloWorld(RESTEntity):
```
Comment (ericvaandering) [inline src/python/WMCore/ReqMgr/Service/Auxiliary.py]: Does this class do anything (used anywhere)?

File: src/python/WMCore/ReqMgr/Service/Auxiliary.py [hunk 3]
```diff
@@ -78,6 +80,90 @@ def get(self, name):
...
+        
+        """
+        validate_str("name", param, safe, rx.RX_REQUEST_NAME, optional=False)
+
+
+    @restcall(formats = [('application/json', JSONFormat())])
+    @tools.expires(secs=-1)
+    def get(self, name):
+        """
+        Hello world API call.
```
Comment (ericvaandering) [inline src/python/WMCore/ReqMgr/Service/Auxiliary.py]: Seems to be the wrong doctoring

File: src/python/WMCore/ReqMgr/Service/Auxiliary.py [hunk 4]
```diff
@@ -78,6 +80,90 @@ def get(self, name):
...
+        
+        helper = WMWorkloadHelper()
+        try:
+            helper.loadSpecFromCouch(self.reqdb_url, name)
+        except Exception:
+            raise cherrypy.HTTPError(404, "Cannot find workload: % "+ name)
+        
+        splittingDict = helper.listJobSplittingParametersByTask(performance = False)
+        taskNames = splittingDict.keys()
+        taskNames.sort()
```
Comment (ericvaandering) [inline src/python/WMCore/ReqMgr/Service/Auxiliary.py]: You can do this on one line with sorted()

File: src/python/WMCore/ReqMgr/Service/Auxiliary.py (hunk 2 above)
Comment (ticoann) [inline src/python/WMCore/ReqMgr/Service/Auxiliary.py]: It was just for the test. I will remove that.

File: src/python/WMCore/ReqMgr/Service/Auxiliary.py (hunk 3 above)
Comment (ticoann) [inline src/python/WMCore/ReqMgr/Service/Auxiliary.py]: Thanks, I will update all the comments

**Comment by ticoann** [comment]
Tested in vm, merging it.

**Pull request merged**